cognite/client/_api_client.py

```diff
@@ -268,7 +268,7 @@ def _list_generator(
         limit: int = None,
         chunk_size: int = None,
         filter: Dict = None,
-        other_params: Dict = None,
+        other_params: Dict = {},
```

Comment:
Please don't use mutable values as defaults. I realize that we don't actually mutate the value so it won't matter here, but it's still good practice to avoid this.